vitest.config.ts

```diff
@@ -0,0 +1,19 @@
+import path from 'path'
+import type { UserConfig } from 'vite'
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    environment: 'jsdom',
+    include: ['packages/**/*.{test,spec}.{js,ts,jsx,tsx}'],
+  },
+  resolve: {
+    alias: {
+      '@tiptap/extension-bold': path.join(__dirname, 'packages/extension-bold/src'),
```

Comment:
The alias resolution is needed so the `Vitest` runner knows how to resolve the `@tiptap/extension-bold`, `@tiptap/extension-document`, and other import aliases correctly

Reply:
Seems verbose, I'd have thought there was a way to resolve using a tsconfig

Reply:
I also ran into a similar issue with a private project and didn't find a quick way to resolve from a tsconfig. There are some posts out there talking about getting it from the tsconfig and also a vite plugin.

However I think what we could do is just parse package names and paths via the filesystem to automate this and have control over it.